config/application.rb

```diff
@@ -39,6 +40,23 @@ class Application < Rails::Application
     # config.time_zone = "Central Time (US & Canada)"
     # config.eager_load_paths << Rails.root.join("extras")
 
+    # RAILS 7 CONFIG START
+    config.action_controller.raise_on_open_redirects = false
```

Comment:
This was defaulted in `true` in 7.0 ([here](https://guides.rubyonrails.org/configuring.html#config-action-controller-raise-on-open-redirects)). Some discussion [here](https://github.com/lynndylanhurley/devise_token_auth/issues/1536) gave some approaches on how to approach the resulting errors with redirects. Talking to @bosawt, we decided we could safely set the default back to `false`.